src/library/OverflowContainer/styled.js

```diff
@@ -31,7 +31,8 @@ export const OverflowContainerRoot = styled('div')(
 
       /* TargetX Custom Styles */
       ...(border && { border }),
-      ...(borderRadius && { borderRadius })
+      ...(borderRadius && { borderRadius }),
+      ...(fullWidth && { width: '100%' })
```

Comment:
@tx-steven  Let's stick to our convention... if we introduce an "alias" we also support the full passthrough. So we should add both `fullWidth` (the alias) and `width` (the passthrough).